peril.settings.json

```diff
@@ -3,7 +3,7 @@
     "onlyForOrgMembers": false
   },
   "rules": {
-    "pull_request.closed": [
+    "pull_request.closed(pull_request.state == "merged")": [
```

Comment:
This is a new feature... first time using it so 🤷‍♂️

Reply:
It'd be great if this works. I tried `.merged = true` and that way didn't work out for us. 😅